package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.0.1",
   "description": "Coding Train Website",
   "main": "none",
-  "dependencies": {},
+  "dependencies": {
+    "replace-in-file": "^3.4.4"
```

Comment:
Perhaps this should be a dev dep instead of a dependency?

Reply:
Agreed!